spec/calculators/assessmentResultCalculatorSpec.js

```diff
@@ -1,18 +1,8 @@
-var Jed = require( "jed" );
 var AssessmentResultCalculator = require( "../../js/calculators/assessmentResultCalculator.js" );
 var MissingArgument = require( "../../js/errors/missingArgument.js" );
 
-var constructi18n = function() {
-	var defaultTranslations = {
-		"domain": "js-text-analysis",
-		"locale_data": {
-			"js-text-analysis": {
-				"": {}
-			}
-		}
-	};
-	return new Jed( defaultTranslations );
-};
+var factory = require( "../helpers/factory.js" );
```

Comment:
This file should not be used anymore. Remove references after https://github.com/Yoast/YoastSEO.js/pull/321 has been merged.